includes/Exception/FailedToMakeInstance.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Exception FailedToMakeInstance.
+ *
+ * @package Google\Web_Stories
+ */
```

Comment:
The regular file description docblocks in the plugiin look more like this

```php
/**
 * Exception FailedToMakeInstance.
 *
 * @package   Google\Web_Stories
 * @copyright 2020 Google LLC
 * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
 * @link      https://github.com/google/web-stories-wp
 */
```

Reply:
Fixed